clinica/pydra/pet_volume/tasks.py

```diff
@@ -0,0 +1,110 @@
+import typing as ty
+from pathlib import PurePath
+
+from pydra.mark import annotate, task
+
```

Comment:
More of an open question here, and not a suggestion/comment. The docstring is very light for the functions in this file. Is it intended as we are only creating a task out of a function, which should be documented ? (and may very well not be at this time).

Reply:
These functions are not supposed to implement anything, they are simple wrappers in order to turn the original functions into Pydra tasks.
Because of this, the doctstrings would basically be copy pasted which I don't think is a good solution because they will be harder to maintain and they will diverge at some point as people forget to maintain them in synchrony.

In [63fc0b1](https://github.com/aramis-lab/clinica/pull/729/commits/63fc0b18545707f962524b79b7a30405dc219b8d), I proposed to add a note encouraging the user to refer to the original function documentation if needed.
Let me know what you think of this possible solution.

Reply:
> These functions are not supposed to implement anything, they are simple wrappers in order to turn the original functions into Pydra tasks.
Because of this, the doctstrings would basically be copy pasted which I don't think is a good solution because they will be harder to maintain and they will diverge at some point as people forget to maintain them in synchrony.

Yup makes sense. If i understand correctly, we'd have to move the documentation once we move on from nipype (which is the reason for which you've 'only' wrapped the function ?)

> Let me know what you think of this possible solution.

At least for know, It looks like a good solution ! Maybe even after we've moved on from nipype :)

Reply:
>If i understand correctly, we'd have to move the documentation once we move on from nipype (which is the reason for which you've 'only' wrapped the function ?)

Either move the docs or keep the function implementation separated from the Pydra task. The point is to avoid duplication (of the code and the docs).